src/makeRobotAccountant.js

```diff
@@ -1,12 +1,32 @@
 'use strict';
 
 /**
- *
+ * function(num1)(num2) === num1 + num2;
  * @return {function}
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let firstNumber;
+  let callNumber = 0;
+
+  const add = function(number = 0) {
+    if (firstNumber === undefined) {
+      firstNumber = number;
+
+      return add;
+    }
+
+    const sum = firstNumber + number;
+
+    firstNumber = undefined;
+    callNumber++;
+
+    return callNumber > 3 && callNumber % 2 === 0
+      ? 'Bzzz... Error!'
+      : sum;
+  };
+
+  return add;
```

Comment:
```suggestion
  let counter = 3;

  retunr (firstValue) => {
    return (secondVlue) => {
      if (counter === 0) {
        errorCounter++;

        return 'Bzzz... Error!';
      }

      errorCounter--;

      return firstValue + secondVlue;
    };
  };
 ```

Reply:
It's not a problem. I usually look through other students' works after I complete a task to gather different ideas and approaches. 
I've seen this one. Ufortunately I could not find this solution from the very begining.
In the end, I left my code to get comments on my solution if it is wrong or if it is worse in some way.